core-active-state.html

```diff
@@ -0,0 +1,101 @@
+<!--
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<link rel="import" href="../polymer/polymer.html">
+<dom-module id="core-active-state"></dom-module>
+<script>
+  Polymer({
+    is: 'core-active-state',
+
+    properties: {
+      active: {
+        type: Boolean,
+        notify: true,
+        readOnly: true,
+        observer: 'activeChanged',
+        value: false
+      },
+
+      pressed: {
+        type: Boolean,
+        value: function() {
+          this._setPressed(this.pressed || false);
+        },
+        readOnly: true,
+        observer: 'pressedChanged'
+      },
+
+      toggle: {
+        type: Boolean,
+        value: false
+      },
+
+      disabled: {
+        type: Boolean,
+        observer: 'disabledChanged',
+        value: false
+      },
+
+      boundDownHandler: {
```

Comment:
I see. Using `properties` only to set default values still seems weird to me, vs. doing `this.boundDownHandler = this._downHandler.bind(this)` in `configure` or `ready`, but I can accept that functionally there's no difference.